app/presenters/compute_resource_presenter.rb

```diff
@@ -0,0 +1,93 @@
+class ComputeResourcePresenter
+  def self.for_cr(compute_resource, view)
+    klass = "#{compute_resource.provider.camelcase}Presenter".safe_constantize
+    Rails.logger.warn("Do not know how to present #{compute_resource.provider}") unless klass
```

Comment:
We should decide if warn about non existance compute resources, or just roll with the default behaviour.

Reply:
since the default presenter can handle the situation, I'd say it shouldn't warn, it may be useful to log in debug log level though